test/lib/services/projectService.test.js

```diff
@@ -0,0 +1,57 @@
+describe('ProjectService', () => {
+    beforeEach(() => {
+        jest.resetModules();
+        console.log = jest.fn(); // create a new mock function for each test
+    })
+    test('createNewProjectPositive', async() => {
+        const { createNewProject } = require('@generate/lib/services/projectService');
+        const { exec } = require('child_process');
+        const projectInformation = {
+            projectType: '',
```

Comment:
fill any relevant information, or example of the information, you can refer to the default value in the code

Reply:
done